CollapseLauncher/XAMLs/MainApp/Pages/SettingsPage.xaml.cs

```diff
@@ -76,6 +77,28 @@ private async void RelocateFolder(object sender, RoutedEventArgs e)
             }
         }
 
+        private async void ClearMetadataFolder(object sender, RoutedEventArgs e)
+        {
+            switch (await Dialog_ClearMetadata(Content))
+            {
+                case ContentDialogResult.Primary:
+                    try
+                    {
+                        Directory.Delete(AppGameConfigMetadataFolder, true);
+                        var collapsePath = Process.GetCurrentProcess().MainModule.FileName;
+                        Process.Start(collapsePath);
+                        App.Current.Exit();
+                    }
+                    catch (Exception ex) 
+                    {
+                        LogWriteLine($"An error occurred!\r\n{ex}", LogType.Error, true);
```

Comment:
`"An error occurred while attempting to clear metadata folder. Exception stacktrace below:\r\n{ex}"`